Source/SA/RFmx SpecAn/RFmxSpecAn.cs

```diff
@@ -11,66 +12,334 @@ public struct CommonConfiguration
+            public RFmxSpecAnMXAcpOffsetSideband OffsetSideBand;
+            public RFmxSpecAnMXAcpOffsetPowerReferenceCarrier OffsetPowerReferenceCarrier;
+            public int OffsetPowerReferenceSpecificIndex;
+            public double RelativeAttenuation_dB;
+            public RFmxSpecAnMXAcpOffsetRrcFilterEnabled RrcFilterEnabled;
+            public double RrcAlpha;
+
+            public static AcpOffsetChannelConfiguration GetDefault()
+            {
+                return new AcpOffsetChannelConfiguration
+                {
+                    IntegrationBandwidth_Hz = 18e6,
+                    OffsetEnabled = RFmxSpecAnMXAcpOffsetEnabled.True,
+                    OffsetFrequency_Hz = 20e6,
+                    OffsetSideBand = RFmxSpecAnMXAcpOffsetSideband.Both,
+                    OffsetPowerReferenceCarrier = RFmxSpecAnMXAcpOffsetPowerReferenceCarrier.Closest,
+                    OffsetPowerReferenceSpecificIndex = 0,
+                    RelativeAttenuation_dB = 0,
+                    RrcFilterEnabled = RFmxSpecAnMXAcpOffsetRrcFilterEnabled.False,
+                    RrcAlpha = 0.22
+                };
+            }
+        }
+        public struct AcpCommonConfiguration
+        {
+            public RFmxSpecAnMXAcpPowerUnits PowerUnits;
+            public RFmxSpecAnMXAcpAveragingEnabled AveragingEnabled;
+            public int AveragingCount;
+            public RFmxSpecAnMXAcpAveragingType AveragingType;
+            public RFmxSpecAnMXAcpFftWindow FftWindow;
+            public double FftPadding;
+
+            public RFmxSpecAnMXAcpRbwAutoBandwidth RbwAuto;
+            public double Rbw_Hz;
+            public RFmxSpecAnMXAcpRbwFilterType RbwFilterType;
+
+            public RFmxSpecAnMXAcpSweepTimeAuto SweepTimeAuto;
+            public double SweepTimeInterval_s;
+            public AcpCarrierChannelConfiguration[] CarrierChannelConfiguration;
+            public AcpOffsetChannelConfiguration[] OffsetChannelConfiguration;
+            public static AcpCommonConfiguration GetDefault()
+            {
+                return new AcpCommonConfiguration
+                {
+                    PowerUnits = RFmxSpecAnMXAcpPowerUnits.dBm,
+                    AveragingEnabled = RFmxSpecAnMXAcpAveragingEnabled.False,
+                    AveragingCount = 10,
+                    AveragingType = RFmxSpecAnMXAcpAveragingType.Rms,
+                    FftWindow = RFmxSpecAnMXAcpFftWindow.FlatTop,
+                    FftPadding = 1,
+                    RbwAuto = RFmxSpecAnMXAcpRbwAutoBandwidth.True,
+                    Rbw_Hz = 10e3,
+                    RbwFilterType = RFmxSpecAnMXAcpRbwFilterType.Gaussian,
+                    SweepTimeAuto = RFmxSpecAnMXAcpSweepTimeAuto.True,
+                    SweepTimeInterval_s = 1e-3,
+                    CarrierChannelConfiguration = new AcpCarrierChannelConfiguration[] { },
+                    OffsetChannelConfiguration = new AcpOffsetChannelConfiguration[] { },

```

Comment:
Discussed in meeting - change to default values rather than empty

Reply:
ensured to have default array values on LV and .net